vscode/src/copilot/webview/hljs-qsharp.ts

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { HLJSApi, Mode, type Language } from "highlight.js";
+
+// NOTE: This has a bunch of differences from how we structure the language in <vscode/syntaxes/qsharp.tmLanguage.json>
+// We should align with that as much as possible eventually.
+
+const KEYWORDS_GENERAL = [
+  "use",
+  "borrow",
+  "mutable",
+  "let",
+  "set",
+  "if",
+  "elif",
+  "else",
+  "repeat",
+  "until",
+  "fixup",
+  "for",
+  "in",
+  "while",
+  "return",
+  "fail",
+  "within",
+  "apply",
+];
+
+const KEYWORDS_DECL = [
+  "namespace",
+  "open",
+  "import",
+  "export",
+  "as",
+  "internal",
+  "newtype",
+  "struct",
+  "operation",
+  "function",
+  "new",
+  "body",
+  "adjoint",
+  "Adjoint",
+  "controlled",
+  "Controlled",
```

Comment:
I'd suggest putting capital `Adjoint` and `Controlled` under a different category, maybe "general". For one, they're not declaration keywords, they're operators. For two, I find it too easy to get mixed up between the lowercase/uppercase versions, and if they have different coloring that may at least be another visual hint that they're different.

Reply:
Copied from our VS Code syntax. Would rather keep consistent for now <https://github.com/microsoft/qsharp/blob/main/vscode/syntaxes/qsharp.tmLanguage.json#L149>